app/Models/ExperienceAward.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace App\Models;
+
+use App\Models\BaseModel;
+
+/**
+ * Class ExperienceAward
+ *
+ * @property int $id
+ * @property string $title
+ * @property int $award_recipient_type_id
+ * @property string $issued_by
+ * @property int $award_recognition_type_id
+ * @property \Jenssegers\Date\Date $awarded_date
+ * @property int $experienceable_id
+ * @property string $experienceable_type
+ * @property \Jenssegers\Date\Date $created_at
+ * @property \Jenssegers\Date\Date $updated_at
+ *
+ * @property \App\Models\Applicant|\App\Models\JobApplication $experienceable
+ */
+class ExperienceAward extends BaseModel
+{
+    protected $casts = [
+        'title' => 'string',
+        'award_recipient_type_id' => 'int',
+        'issued_by' => 'string',
+        'award_recognition_type_id' => 'int',
+        'awarded_date' => 'date'
+    ];
+
+    protected $fillable = [
+        'title',
+        'award_recipient_type_id',
+        'issued_by',
+        'award_recognition_type_id',
+        'awarded_date'
+    ];
+
+    protected $table = ['experiences_award'];
```

Comment:
This should just be a string, not array. https://laravel.com/docs/5.8/eloquent#eloquent-model-conventions

Reply:
I just tested it, looks like it works anyway.

Reply:
I fixed this on #3018.